mopidy_tidal/library.py

```diff
@@ -69,69 +62,65 @@ def _get_image_uri(cls, obj):
         cls._log_image_not_found(obj)
 
     def _get_api_getter(self, item_type: str):
-        tidal_lt_0_7_getter_name = f'get_{item_type}'
+        tidal_lt_0_7_getter_name = f"get_{item_type}"
+        # TODO: can't we use nested getattr here?
```

Comment:
Let's at an issue to fix this.

Reply:
I've just committed it. Meant to do that when the test suite was in place to prove nothing was broken.